pkg/operator/ceph/config/conditions.go

```diff
@@ -47,23 +41,19 @@ func ConditionExport(context *clusterd.Context, namespaceName types.NamespacedNa
 
 // setCondition updates the conditions of the cluster custom resource
 func setCondition(c *clusterd.Context, namespaceName types.NamespacedName, newCondition cephv1.Condition) {
-	ctx := context.TODO()
-	cluster, err := c.RookClientset.CephV1().CephClusters(namespaceName.Namespace).Get(ctx, namespaceName.Name, metav1.GetOptions{})
+	cluster, err := getCluster(c, namespaceName)
 	if err != nil {
-		if kerrors.IsNotFound(err) {
-			logger.Errorf("no CephCluster could not be found. %+v", err)
-			return
-		}
-		logger.Errorf("failed to get CephCluster object. %+v", err)
```

Comment:
We still should log an error so we know the conditions were not updated as expected.

Reply:
The error is already being logged in `getCluster()`. Since that is an internal function, I did not think it needed to be logged again.

Reply:
Ah, I overlooked that. So we will have double logging on this error since I was trigger happy with the merge. The pattern more common in rook would be to only log at the point where the error is ignored, so it's clear that it's intentionally ignored.